src/kirby/components/badge/badge.component.html

```diff
@@ -0,0 +1,5 @@
+<ion-badge [color]="colortype">
```

Comment:
Probably need to add:
```suggestion
<ion-badge mode="ios" [color]="colortype">
```
to make it consistent with the UX design / across platforms